app/src/main/java/cm/aptoide/pt/home/apps/AppsAdapter.java

```diff
@@ -321,4 +321,27 @@ public void setAvailableUpdatesList(List<App> availableUpdates) {
     updatesToRemove.removeAll(updatesList);
     return updatesToRemove;
   }
+
+  public void setAppIndeterminate(App app) {
+    int indexOfApp = listOfApps.indexOf(app);
+    if (indexOfApp != -1) {
+      App application = listOfApps.get(indexOfApp);
+      if (application.getType() == App.Type.UPDATE) {
+        setIndeterminate(indexOfApp, (UpdateApp) application);
+      }
+    }
+  }
+
+  private void setIndeterminate(int indexOfApp, UpdateApp application) {
```

Comment:
naming: Maybe setIndeterminateStatus

Reply:
Don't know if status would be correct term since the status that is being set is the STANDBY. Also, this attribute is being used to set the indeterminate on the progress bar, whose method is also called setIndeterminate, that is why i decided to name it the same way.